app/retail/templates/presskit.html

```diff
@@ -26,7 +26,6 @@
     <div class="content__main min-vh-100 d-md-flex flex-column">
       {% include 'shared/tag_manager_2.html' %}
       <div class="header">
-        {% include 'shared/top_nav.html' with class='d-md-flex' %}
```

Comment:
This line shouldn't be removed could you add it back in and rebase against `master` please?

Reply:
will add it back now and rebase - thanks for letting me know!